packages/quantic/force-app/main/default/lwc/quanticRecommendationList/__tests__/quanticRecommendationList.test.js

```diff
@@ -0,0 +1,428 @@
+          const element = createTestComponent({
+            ...defaultOptions,
+            variant,
+          });
+          await flushPromises();
+
+          const recommendationContainer =
+            element.shadowRoot.querySelector(containerSelector);
+          const recommendationItems =
+            element.shadowRoot.querySelectorAll(itemSelector);
+
+          expect(recommendationContainer).not.toBeNull();
+          if (variant === 'grid') {
+            expect(recommendationItems.length).toBe(
+              recommendationsArray.length
+            );
+          } else {
+            expect(recommendationContainer.label).toEqual(defaultOptions.label);
+            expect(recommendationContainer.itemsPerPage).toEqual(
+              defaultOptions.recommendationsPerRow
+            );
+          }
+        });
+
+        describe('with a custom #recommendationsPerRow option', () => {
+          const customRecommendationsPerRow = 2;
+          if (variant === 'grid') {
+            it('should set the width of the recommendation items based on the number of #recommendationsPerRow value', async () => {
+              const expectedRecommendationWidth = `${100 / customRecommendationsPerRow}%`;
+              const element = createTestComponent({
+                ...defaultOptions,
+                variant,
+                recommendationsPerRow: 2,
+              });
+              await flushPromises();
+
+              expect(
+                element.style._values['--recommendationItemWidth']
+              ).toEqual(expectedRecommendationWidth);
+            });
+          } else {
+            it('should pass the #recommendationsPerRow value to the carousel #itemsPerPage property', async () => {
+              const element = createTestComponent({
+                ...defaultOptions,
+                variant,
+                recommendationsPerRow: customRecommendationsPerRow,
+              });
+              await flushPromises();
+              const carousel = element.shadowRoot.querySelector(
+                selectors.carouselContainer
+              );
+
+              expect(carousel).not.toBeNull();
+              expect(carousel.itemsPerPage).toEqual(
+                customRecommendationsPerRow
+              );
+            });
+          }
+        });
+
+        it('should properly pass the custom #label to the heading', async () => {
+          const customLabel = 'Custom label';
+          const element = createTestComponent({
+            ...defaultOptions,
+            variant,
+            label: customLabel,
+          });
+          await flushPromises();
+
+          const headingElement = element.shadowRoot.querySelector(
+            selectors.quanticHeading
+          );
+
+          expect(headingElement).not.toBeNull();
+          expect(headingElement.label).toEqual(customLabel);
+        });
+      });
+
+      describe('when invalid property values are passed', () => {
```

Comment:
This should belong to the describe  `component initialization` and not the describe `when the component is initialized successfully`. if there is an invalid property it's not a successful initialization.